build.gradle

```diff
@@ -3458,7 +3462,7 @@ project(":web") {
         def targetProperties = project.rootProject.ext[t.upper]
         def webkitProperties = targetProperties.webkit
         def classifier = (t.name != "linux" && t.name != "win") ? t.name :
-                          IS_64 ? "${t.name}-amd64" : "${t.name}-i586"
+                          IS_64 ? IS_AARCH64 ? "${t.name}-aarch64" : "${t.name}-amd64" : "${t.name}-i586"
```

Comment:
I don't think this change is necessary as `classifier` was used for webview-deps, and it's not used anymore (removed in https://github.com/openjdk/jfx/commit/bf203ad716633e8c9f23a70448fac1ed580a7fb5). This two lines are also be removed.

Reply:
that would be a good cleanup indeed.